### PR TITLE
Update to use Suave v1.0

### PR DESCRIPTION
**Pull request by andredublin**

### CODE REVIEW

File: en/views.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ In this application we'll use server-side HTML templating with the help of a sep
 > Note: As of the time of writing, `Suave.Experimental` is a separate package. It's likely that next releases of the package will include breaking changes. It's also possible that the modules we're going to use from within the package will be extracted to the core Suave package.
 
 To use the package, we need to take a dependency on the following NuGet:
-```install-package Suave.Experimental -version 0.28.1```
+```install-package Suave.Experimental```
```
Comment (theimowski) [inline en/views.md]: I'd rather stick with pinned version (`-version 1.0`), to have content synced with code

Comment (andredublin) [inline en/views.md]: I've added that in now

**Comment by theimowski** [comment]
Awesome! Did you indeed check that those changes make it work with Suave 1.0 ? 
[This](https://github.com/theimowski/SuaveMusicStore) repo will have to be updated as well, I still need to come up with a best way to do that.
For now, we'll have to mark that all permalinks to git tags (e.g. [here at the end of file](https://github.com/theimowski/SuaveMusicStoreTutorial/blob/master/en/admin_authorization.md)) show version 0.28.1

**Comment by andredublin** [comment]
Yes, I worked through the book with the updated code and made sure the changes worked.

**Comment by theimowski** [comment]
Cool, can we please temporarily add `(Suave 0.28.1)` after every such git tag permalink? There should be 10 of them for each language. That's just to make the reader less confused

**Comment by andredublin** [comment]
Links added

**Comment by theimowski** [comment]
Thanks!

**Pull request merged**